Pinta.Effects/Dialogs/Effects.AlignmentDialog.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using Gtk;
+using Pinta.Core;
+
+namespace Pinta.Effects;
+
+public sealed class AlignmentDialog : Gtk.Dialog
+{
+	private readonly Gtk.CheckButton topLeft;
+	private readonly Gtk.CheckButton topCenter;
+	private readonly Gtk.CheckButton topRight;
+	private readonly Gtk.CheckButton centerLeft;
+	private readonly Gtk.CheckButton center;
+	private readonly Gtk.CheckButton centerRight;
+	private readonly Gtk.CheckButton bottomLeft;
+	private readonly Gtk.CheckButton bottomCenter;
+	private readonly Gtk.CheckButton bottomRight;
+
+	public AlignPosition SelectedPosition { get; private set; }
+
+	public event EventHandler? PositionChanged;
+
+	public AlignmentDialog (IChromeService chrome)
+	{
+		const int spacing = 6;
+
+		var grid = new Gtk.Grid {
+			RowSpacing = spacing,
+			ColumnSpacing = spacing,
+			RowHomogeneous = true,
+			ColumnHomogeneous = true,
+			MarginStart = 12,
+			MarginEnd = 12,
+			MarginTop = 12,
+			MarginBottom = 12
+		};
+
+		topLeft = CreateCheckButton ("Top Left", AlignPosition.TopLeft);
```

Comment:
These strings labels would need to become e.g. `Translation.GetString("Top Left")` in order for translations to work

One other idea for the UI though would be to use the same alignment buttons as the Resize Canvas dialog - what do you think about that approach? It might be less cluttered if there are other options added in the future, e.g. aligning only horizontally or only vertically


<img width="302" alt="resize_canvas" src="https://github.com/user-attachments/assets/4de32aad-d316-466e-bc20-0e128e1c93f6">

Reply:
Indeed, that was the approach i originally wanted to take, but i didn't know if we had icons we could use for it.
I switched to that, although the center icon isn't that fitting, it should probably be something like a dot / circle.

Reply:
completely slipped my mind :sweat_smile: 
should be good now!

Reply:
I should have been more clear, sorry :) 
Doing `Translations.GetString (tooltip)` isn't exactly right - the problem is that those tooltip strings won't be picked up automatically and added to the translation template (`po/messages.pot`) as strings that can be translated (https://github.com/PintaProject/Pinta/wiki/Translations)

So the approach is to explicitly do `Translations.GetString("Top Left")`, `Translations.GetString("Top Center")` so that those strings are automatically picked up for translations

Reply:
Makes sense, moved the GetString calls to use literals